tests/manage/z_cluster/test_large_omap_objects_fs_meta_data.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import pytest
+
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.ocs.cluster import ceph_health_check
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+)
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier2
+@ignore_leftovers
+@bugzilla("2120944")
+@skipif_external_mode
```

Comment:
This test is not supported on external mode?

Reply:
done